File: 0387-first-unique-character-in-a-string/0387-first-unique-character-in-a-string.java
class Solution {
    public int firstUniqChar(String s) {
       char[] c=s.toCharArray();

       for(int i=0;i<c.length;i++){
        boolean uni=true;
        
        for(int j=0;j<c.length;j++){
            if(i!=j&&c[i]==c[j]){
            uni=false;
            break;
            }
        }
        if(uni){
            return i;
            
        }
    
       }
       return -1;

    }
}

File: 0387-first-unique-character-in-a-string/0387-first-unique-character-in-a-string.c
int firstUniqChar(char* s) {

    for(int i=0;i<strlen(s);i++){
        bool u=true;
        for(int j=0;j<strlen(s);j++){
        if(i!=j&&s[i]==s[j]){
            u=false;
            break;
        }
        }
        if(u)return i;
    }
    return -1;
}